Dockerfile

```diff
@@ -40,20 +65,20 @@ RUN apt-get update && apt-get install --yes --no-install-recommends \
   && ln -sf /usr/bin/nodejs /usr/bin/node \
   && ln -sf /bin/bash /bin/sh
 
-ENV HOME=/home/captain \
-  WSBU_C_COMPILER=/opt/linaro/bin/arm-linux-gnueabihf-gcc \
-  WSBU_CXX_COMPILER=/opt/linaro/bin/arm-linux-gnueabihf-g++ \
-  WSBU_EMULATOR=/usr/bin/qemu-arm \
-  QEMU_LD_PREFIX=/opt/linaro/arm-linux-gnueabihf/libc \
-  CMAKE_TOOLCHAIN_FILE=/opt/toolchain-linaro-armhf.cmake
-
-COPY toolchain.cmake "$CMAKE_TOOLCHAIN_FILE"
-
+ENV GCC_PREFIX=/opt/linaro
 RUN git clone https://github.com/wsbu/linaro-release.git \
     --branch gcc-linaro-4.9.4-2017.01-x86_64_arm-linux-gnueabihf \
     --depth 1 \
-    /opt/linaro \
-  && rm -rf /opt/linaro/.git
+    "${GCC_PREFIX}" \
+  && rm -rf "${GCC_PREFIX}/.git"
+
+ENV HOME=/home/captain \
+  WSBU_C_COMPILER=${GCC_PREFIX}/bin/arm-linux-gnueabihf-gcc \
+  WSBU_CXX_COMPILER=${GCC_PREFIX}/bin/arm-linux-gnueabihf-g++ \
+  WSBU_EMULATOR=/usr/bin/qemu-arm \
+  QEMU_LD_PREFIX=${GCC_PREFIX}/arm-linux-gnueabihf/libc \
+  CMAKE_TOOLCHAIN_FILE=/opt/toolchain-linaro-armhf.cmake
+COPY toolchain.cmake "${CMAKE_TOOLCHAIN_FILE}"
```

Comment:
Why did you drop the space here? I find it easier to read with the space.

Reply:
What space?

Reply:
The one in Paul's original. There was a newline before the COPY directive; at least, that's what the diff showed.

Reply:
Found it. I couldn't see that on my phone for some reason, but it's fixed now (and yes, I'll squash before merging).